parastell/source_mesh.py

```diff
@@ -6,36 +6,56 @@
 import pystell.read_vmec as read_vmec
 
 from . import log as log
-from .utils import read_yaml_config, filter_kwargs, m2cm
+from .utils import read_yaml_config, filter_kwargs, m2cm, m3tocm3
 
 export_allowed_kwargs = ['filename']
 
 
-def rxn_rate(s):
-    """Calculates fusion reaction rate in plasma.
+def default_reaction_rate(n_i, T_i):
+    """Default reaction rate formula for DT fusion
+    assumes an equal mixture of D and T in a hot plasma.
```

Comment:
Do we have a reference for this?

Reply:
See [this paper](https://iopscience.iop.org/article/10.1088/1741-4326/ac2991/meta) by Aaron, on page four. It gives this equation using separate functions for D and T concentrations, but here we assume them to be equal (per Aaron's suggestion).

Reply:
That appears to be based on [this paper](https://www.tandfonline.com/doi/abs/10.13182/FST08-A1899).  Can we confirm what they actually use here and update accordingly?